mod/data/hullmods/hull_mods.csv

```diff
@@ -13,4 +13,6 @@ Order: Retreat when Missiles Expended,automatic_orders_retreat_no_missiles,0,,Au
 Order: Retreat at Malfunction Threshold,automatic_orders_retreat_malfunction,0,,Automatic Orders,,,0,TRUE,,,0,0,0,0,com.github.isturdy.automaticorders.hullmods.RetreatMalfunction,Orders the ship to retreat when it reaches the malfunction threshold. Overrides the default retreat threshold.,Orders the ship to retreat when it reaches the malfunction threshold.,graphics/automatic_orders/hullmods/retreat_malfunction.png
 Order: Retreat at Critical Malfunction Threshold,automatic_orders_retreat_critical,0,,Automatic Orders,,,0,TRUE,,,0,0,0,0,com.github.isturdy.automaticorders.hullmods.RetreatCritical,Orders the ship to retreat when it reaches the critical malfunction threshold. Overrides the default retreat threshold.,Orders the ship to retreat when it reaches the critical malfunction threshold.,graphics/automatic_orders/hullmods/retreat_critical_malfunction.png
 Order: No CR Retreat,automatic_orders_no_cr_retreat,0,,Automatic Orders,,,0,TRUE,,,0,0,0,0,com.github.isturdy.automaticorders.hullmods.NoCrRetreat,Disables automatic retreat at low PPT/CR.,Disables automatic retreat at low PPT/CR.,graphics/automatic_orders/hullmods/no_cr_retreat.png
-Order: No Retreat,automatic_orders_no_retreat,0,,Automatic Orders,,,0,TRUE,,,0,0,0,0,com.github.isturdy.automaticorders.hullmods.NoRetreat,Disables automatic retreat at low PPT/CR or high damage.,Disables automatic retreat at low PPT/CR or high damage.,graphics/automatic_orders/hullmods/no_retreat.png
\ No newline at end of file
+Order: No Retreat,automatic_orders_no_retreat,0,,Automatic Orders,,,0,TRUE,,,0,0,0,0,com.github.isturdy.automaticorders.hullmods.NoRetreat,Disables automatic retreat at low PPT/CR or high damage.,Disables automatic retreat at low PPT/CR or high damage.,graphics/automatic_orders/hullmods/no_retreat.png
+Order: Retreat at 10% PPT,automatic_orders_ten_ppt_retreat,0,,Automatic Orders,,,0,TRUE,,,0,0,0,0,com.github.isturdy.automaticorders.hullmods.RetreatTenPPT,Orders the ship to retreat when it reaches the 10% of its maximum Peak Performance Time. Overrides the default retreat threshold.,Orders the ship to retreat when it reaches 10% PPT threshold.,graphics/automatic_orders/hullmods/retreat_ten_ppt.png
```

Comment:
Nit: can you make these follow the "automatic_orders_retreat_{THRESHOLD}" convention?

Reply:
as is tradition.
followed and resolved